common/script/fns/predictableRandomNamespaced.js

```diff
@@ -0,0 +1,5 @@
+import predictableRandom from './predictableRandom';
```

Comment:
I don't like this pattern. We should either export predictableRandom like this in the first place or attach all the functions to an object and import them that way whenever they are used.